developing/languages/ruby/installation.md

```diff
@@ -0,0 +1,84 @@
+# [Ruby](./README.md) / Installation
+
+## Context
+
+Installing and configuring Ruby on a Mac is a common source of confusion and
+frustration for folks of all experience levels due to inconsistent and
+incomplete instructions found online.
+
+Although macOS ships with Ruby, it is not recommended to use it for development
+for several reasons:
+
+* It is usually an older version, and is only updated as part of a major macOS
+release.
+
+* Installing gems in the system Ruby requires the use of `sudo`. In general, it
+should be assumed that everything under `/System/Library` is part of macOS and
+is administered by Apple. Files in those directories should not be modified.
+
+* [macOS will no longer include Ruby by default in future releases](https://developer.apple.com/documentation/macos_release_notes/macos_catalina_10_15_beta_9_release_notes).
+
+## Approaches
+
+### Limited to one version of Ruby at a time
+
+* Install Ruby with homebrew
+
+* Install Ruby manually
+
+### Allows switching between multiple versions of Ruby
+
+* Install and configure a Ruby manager, then install Ruby
+
+* Use a vetted automated script that sets everything up for you
+
+## The Trussel Way
+
+For long-term day-to-day development, we recommend installing and using Ruby via
+a Ruby manager. Inevitably, when working with Ruby projects, you will need to
+install multiple different versions at the same time and switch between them.
+
+Depending on your preferences and needs, the two recommended ways to install and
+use Ruby on a Mac are via local installation of a Ruby manager, or via Docker.
+
+### Local installation
+
+The easiest and most reliable way to install Ruby locally is via an automated
+[script] which installs the following tools (only if you don't already have them):
+
+* [Homebrew](https://brew.sh/)
+* Apple Command Line Tools (note that you do not need the full Xcode package)
+* [chruby](https://github.com/postmodern/chruby) (our recommended Ruby manager)
+* [ruby-install](https://github.com/postmodern/ruby-install) (chruby's companion
+tool for installing different Ruby versions)
+* The latest version of Ruby at the time you run the script
+* [bundler](https://bundler.io/)
+
+The script also updates your shell startup file (such as `.bash_profile`) to
+properly configure `chruby`.
+
+[script]: https://github.com/monfresh/install-ruby-on-macos
+
+### Docker installation
+
+WIP
```

Comment:
I vote either not including this section or putting an example of

`docker run -it --rm=true -v $PWD:$PWD -w $PWD ruby:${DESIRED_RUBY_VERSION}-slim bash`

Reply:
I expanded the Docker section here: https://github.com/trussworks/Engineering-Playbook/pull/43/commits/c694925782d00632b69e0699701b6b941993dd08

Thoughts?

Reply:
Awesome!  We do have a docker section in the handbook, maybe link to that for the installation instructions?

Reply:
Great idea! I missed that.

Reply:
Fixed here: https://github.com/trussworks/Engineering-Playbook/pull/43/commits/7d2213e27b75330b77b19c7d7f47a2c9e917d67f